src/amo/pages/AddonInfo/styles.scss

```diff
@@ -18,3 +18,17 @@
     width: 65%;
   }
 }
+
+.AddonMoreInfo-tag-links-list {
+  li {
+    display: inline-block;
+  }
+
+  li::after {
+    content: '\00a0\2022\00a0';
```

Comment:
Is this the small bullet? Other than that, I think the only concern is to make sure we aren't downloading font files unnecessarily.

Reply:
it wasn't, it was the large bullet - I was waiting to see what Jorge thought.  Changed in https://github.com/mozilla/addons-frontend/pull/10798/commits/40f037576d4effa767f6211c8deabd8b36da461d

Reply:
\2219 isn't in either font subset (the restricted or full) so locally it seems to be fulfilled by a system font (Arial or Segoe for me, depending on the locale).